### DISCUSSION TITLE
[JUNO]PE击中直方图的横纵坐标应该是什么

### DISCUSSION THREAD

**Discussion by CreatinGL** [physics-data/faq#161] [Q&A] [answered]
请问助教老师，最后要求输出图像中的"PE击中直方图"的横纵坐标应该是什么？是对一个PMT而言还是所有PMT。

**Comment by yuchenguommm** [comment, marked as answer]
我想这应该不是很重要?因为不同的PMT之间具有旋转对称性。我个人选择的是所有PMT，这样作出的图经过了PMT的平均后更加好看。

**Reply by heroxbd** [reply]
楼上正解，谢谢！
